android/src/test/java/dev/openfeature/sdk/DeveloperExperienceTests.kt

```diff
@@ -95,7 +96,7 @@ class DeveloperExperienceTests {
         val dispatcher = StandardTestDispatcher(testScheduler)
         var eventCount = 0
         CoroutineScope(dispatcher).launch {
-            OpenFeatureAPI.observeEvents().collect {
+            OpenFeatureAPI.addHandler<OpenFeatureEvents.ProviderReady>().collect {
```

Comment:
The intention behind this change is to bring the API design as close as possible to the [Spec](https://openfeature.dev/docs/reference/concepts/events/#event-handlers)